modules/ROOT/pages/migration-prep.adoc

```diff
@@ -45,11 +44,14 @@ Next, you need to make sure your deployment environment is ready. Depending on w
 |Hybrid Deployment
 |You must install the standalone version of Mule 4 your servers.
 
+|Anypoint Runtime Fabric
+|Runtime Fabric is Mule 4 ready.
+
 |Anypoint Platform Private Cloud Edition (PCE)
-| PCE supports Mule runtime, version 4.
+| PCE supports Mule Runtime 4
 
 |Anypoint Platform for Pivotal Cloud Foundry (PCF)
-| PCF does not support Mule runtime, version 4.
+| PCF does not support Mule Runtime 4
```

Comment:
```suggestion
| PCF does not support Mule 4.
```

Reply:
...or in all of the above cases, you can substitute "Mule runtime engine 4/v4/version 4, if you prefer.